release/models/policy-forwarding/openconfig-pf-forwarding-policies.yang

```diff
@@ -252,6 +277,19 @@ submodule openconfig-pf-forwarding-policies {
     }
   }
 
+  grouping pf-forwarding-policy-rule-ni-config {
+    description
+      "Forwarding policy rules for network-instance matching.";
+
+    leaf input-match-all {
+      type boolean;
+      default false;
+      description
+        "If set to true, all packets received on a network instance
```

Comment:
"all received on a network instance" is ambiguous.
Does this include only the packets arriving on interfaces attached to the network-instance?

Or is there an assumption that this statement also covers other use cases (if so, which)? Different scenarios are possible, such as:
- packets that are redirected via another PBF rule for the next lookup
- packets arriving with encapsulation headers (e.g. mpls or vxlan) that indicate the NI for the next lookup/action,
- vrf leaking

Reply:
For the use case we have in mind, matching all IPv4/6 packets is sufficient, so using a prefix match rules for 0.0.0.0/0 and ::/0  works.   This change is not needed.